src/pages/index.astro

```diff
@@ -183,217 +176,66 @@ const metadata = {
     isAfterContent
     items={[
       {
-        title: 'Enhanced user engagement',
+        title: 'Series A Funding Goals',
```

Comment:
Nit: Hahaha. This is not the appropriate place to put fundraising goals, but that's ok this is just the first pass.